app/config.py

```diff
@@ -127,6 +127,7 @@ class Config(object):
     SQLALCHEMY_COMMIT_ON_TEARDOWN = False
     SQLALCHEMY_RECORD_QUERIES = True
     SQLALCHEMY_TRACK_MODIFICATIONS = True
+    SQLALCHEMY_POOL_SIZE = os.environ.get('SQLALCHEMY_POOL_SIZE', 5)
```

Comment:
I think we should set SQLALCHEMY_MAX_OVERFLOW as well?

If the overflow is 10 and the pool size if 10 we are technically no better off? Its just more efficient as the overflow connections don't stay open whereas the pool connection will.

Maybe set it to SQLALCHEMY_MAX_OVERFLOW = 15?

Reply:
Why are we not better off with the default overflow of 10? 

Before we would go to 15 connections per worker per instance (5 permanent + 10 temporary) and now we will go to 20 connections per worker per instance (10 permanent + 10 temporary).

We have double the amount of permanent connections and we keep the same headroom for extra connections if necessary. Am I missing something?